ebos/equil/initstateequil.cc

```diff
@@ -1901,12 +1906,29 @@ using GridView = Dune::Fem::GridPart2GridViewImpl<
 using GridView = Dune::GridView<Dune::DefaultLeafGridViewTraits<Dune::CpGrid>>;
 #endif
 
+#if HAVE_DUNE_ALUGRID
+
+#if HAVE_MPI
+    using ALUGrid3CN = Dune::ALUGrid<3, 3, Dune::cube, Dune::nonconforming, Dune::ALUGridMPIComm>;
+#else
+    using ALUGrid3CN = Dune::ALUGrid<3, 3, Dune::cube, Dune::nonconforming, Dune::ALUGridNoComm>;
+#endif //HAVE_MPI
+#endif //HAVE_DUNE_ALUGRID
```

Comment:
Any reason not to put this together with the other `HAVE_DUNE_ALUGRID` section&mdash;i.e., the one that instantiates the `InitialStateComputer<>` template for ALUGrid?  On a separate note, I might actually write this bit as
```C++
#if HAVE_MPI
using ALUGridComm = Dune::ALUGridMPIComm;
#else
using ALUGridComm = Dune::ALUGridNoComm;
#endif // HAVE_MPI

using ALUGrid3CN = Dune::ALUGrid<3, 3, Dune::cube, Dune::nonconforming, ALUGridComm>;
```
to make it visually clear that the only difference between the two alternatives is the communicator.